### PR TITLE
Pass JS var as a function param instead of a global var

### LINKED ISSUE #138: Prevent Javascript errors for overridden Order details page
# Abstract

The themes installed by the user can override the CSS and JS from the modules installed. 

# Details

To avoid "variable not found" errors on our front-office JS, we must not pass the variables to the JS by a script tag in the template

# Requester

client

### PR DESCRIPTION
**Pull request by PaulGirardin**
-----
fixes #138 (automatic)
fixed shoprunback/kanban#97

### CODE REVIEW

File: views/templates/front/orderDetail.tpl [hunk 1]
```diff
@@ -22,7 +18,13 @@
                     {l s='front.order.detail.modal.cancel' mod='shoprunback'}
                 </span>
 
-                <span onclick="createShopRunBackReturn('{$createReturnLink nofilter}')" class="external-link form-control-submit btn btn-primary">
+                <span
+                    onclick="createShopRunBackReturn(
+                        '{$createReturnLink nofilter}',
+                        &quot;<h1>{l s='front.order.detail.loading' mod='shoprunback'}</h1>&quot;
```
Comment (tildedash) [inline views/templates/front/orderDetail.tpl]: &quot ?

Comment (leo-martin) [reply views/templates/front/orderDetail.tpl]: used to add a " in a html file, we cannot escape double quotes

**Comment by leo-martin** [comment]
Tested on 1.7.2.1 : OK

**Comment by leo-martin** [comment]
Tested on 1.6.0.9 (deprecated by PS) : OK

**Comment by leo-martin** [comment]
1.0.16

**Pull request merged**